P/pprof_jll/Versions.toml

```diff
@@ -10,8 +10,5 @@ git-tree-sha1 = "6c056f2c6463c58a7083c1294710cf6ab7de697e"
 ["1.0.0+0"]
 git-tree-sha1 = "4a6859c2ffb30b8d6cb4ac73d2b6fcdd945c60c2"
 
-["1.0.0+1"]
-git-tree-sha1 = "d1a2f409c8573751f89f29065498f467ebc87785"
-
```

Comment:
This isn't yanking.
```toml
["1.0.0+1"]
git-tree-sha1 = "d1a2f409c8573751f89f29065498f467ebc87785"
yanked = true
```
would be yanking.

Reply:
oops!

Reply:
ok, pushed a new version